compiler-core/src/metadata/module_decoder.rs

```diff
@@ -75,6 +75,8 @@ impl ModuleDecoder {
             ),
             values: read_hashmap!(reader.get_values()?, self, value_constructor),
             accessors: read_hashmap!(reader.get_accessors()?, self, accessors_map),
+            // Unused locations are not serialized
+            unused: Vec::new(),
```

Comment:
How come?

Reply:
I wasn't sure if they needed to be serialized. I've updated the schema in the last commit.